### ISSUE TITLE
Event Handler Error

### ISSUE THREAD

**Issue by emergingdzns** [GeneaLabs/laravel-mixpanel#4]
Now that it's installed, before adding anything to controllers or anything (only did the composer install and added the provider row to the app config file providers array.

Now I'm getting the following error:
```
FatalErrorException in LaravelMixpanelEventHandler.php line 63:
Call to a member function format() on null
```

Any ideas?

**Comment by mikebronner** [comment]
This appears to be a result of nat having timestamps on your users table -- is that a correct assumption? The created_at field (if it exists) appears to be null. I will add a check for that to prevent the error. Can you do the following just before line 63 and after line 62 to test: 

```php
dd($user);
```

**Comment by mikebronner** [comment]
If you can confirm that, I will post a new release so you can run `composer update` to get the fix. :) Thanks!

**Comment by emergingdzns** [comment]
I can't even find that file man. Where is it supposed to be located?

**Comment by mikebronner** [comment]
Should be in `/vendor/genealabs/laravel-mixpanel/src/Listeners/LaravelMixpanelEventHandler.php`

**Comment by emergingdzns** [comment]
User {#237 ▼
  #primaryKey: "user_id"
  #table: "users"
  +timestamps: false
  -visibleTwitterPredicateCategories: null
  -visibleTwitterTopics: null
  -visibleNPI: null
  #connection: null
  #perPage: 15
  +incrementing: true
  #attributes: array:10 [▶]
  #original: array:8 [▶]
  #relations: []
  #hidden: []
  #visible: []
  #appends: []
  #fillable: []
  #guarded: array:1 [▶]
  #dates: []
  #dateFormat: null
  #casts: []
  #touches: []
  #observables: []
  #with: []
  #morphClass: null
  +exists: true
  +wasRecentlyCreated: false
}
Yeah we don't have a created_at field.

**Comment by mikebronner** [comment]
Perfect, thanks for checking. Just published fix version 0.4.5. Run `composer update` in a few minutes, and this issue should be resolved. Thanks for finding this problem for me! :)

**Comment by emergingdzns** [comment]
Ok that fixed that problem.
